### PR TITLE
S04 display video

### PR DESCRIPTION
**Pull request by Maachino**
Display Video from YT player 
Delete the unused comp

### CODE REVIEW

File: frontend/src/components/VideoComponent.jsx [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
-              kind="captions"
-              src={video.captionsUrl}
-              label="French"
-              default
-            />
-          </video>
-          <figcaption>{video.caption}</figcaption>
-        </figure>
-      ))}
+      <figure key={video.id}>
```
Comment (rocambille) [inline frontend/src/components/VideoComponent.jsx]: you're not in a map : no need of a key

File: frontend/src/components/VideoComponent.jsx [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
-          "Une erreur s'est produite lors de la récupération des vidéos :",
+          "Une erreur s'est produite lors de la récupération de la video :",
           error
         );
       });
-  }, []);
-
+  }, [vidId]);
   return (
     <div>
```
Comment (rocambille) [inline frontend/src/components/VideoComponent.jsx]: remove this div

File: frontend/src/components/VideoComponent.jsx [hunk 3]
```diff
@@ -1,39 +1,41 @@
...
     )
       .then((response) => response.json())
       .then((data) => {
-        setVideos(data);
+        setVideo(data);
       })
       .catch((error) => {
         console.error(
-          "Une erreur s'est produite lors de la récupération des vidéos :",
+          "Une erreur s'est produite lors de la récupération de la video :",
```
Comment (rocambille) [inline frontend/src/components/VideoComponent.jsx]: vidéo, not video

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/VideoComponent.jsx (hunk 1 above)
Comment (Maachino) [reply frontend/src/components/VideoComponent.jsx]: fixed

File: frontend/src/components/VideoComponent.jsx (hunk 2 above)
Comment (Maachino) [reply frontend/src/components/VideoComponent.jsx]: fixed

File: frontend/src/components/VideoComponent.jsx (hunk 3 above)
Comment (Maachino) [reply frontend/src/components/VideoComponent.jsx]: fixed

**Pull request merged**